src/coreclr/debug/di/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ if(CLR_CMAKE_HOST_WIN32)
     endif()
 elseif(CLR_CMAKE_HOST_UNIX)
 
-    if(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_ARM)
+    if(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_ARM OR CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
```

Comment:
I think whole changes in this file can be moved to https://github.com/dotnet/runtime/pull/62886 to make this PR sef-contained.

Reply:
Thanks.
Had moved.